app/javascript/controllers/breadcrumb_controller.js

```diff
@@ -130,6 +133,10 @@ export default class extends Controller {
         if (usedWidth < width) {
           visibleSet.add(i);
         } else {
+          // if at least one item invisible, check that there is enough space for the dropdown menu
+          if (i >= 0 && width - usedWidth < this.dropdownMenuTarget.width) {
+            visibleSet.remove(i + 1);
```

Comment:
This should be `visibleSet.delete(i + 1);`

Reply:
Fixed in 4c8d775